components/scream/src/share/scream_pack.hpp

```diff
@@ -364,6 +364,15 @@ OnlyPack<PackType> pow (const PackType& a, const ScalarType/*&*/ b) {
   return s;
 }
 
+template <typename ScalarType, typename PackType>
+KOKKOS_INLINE_FUNCTION
+OnlyPack<PackType> pow (const ScalarType& a, const PackType/*&*/ b) {
```

Comment:
I'm not 100% sure about the lack of reference for the pack. In the code immediately above, it's the Scalar that was passed by value.

Reply:
Yes, please take `b` by reference.

Please also add a unit test in components/scream/src/share/tests/pack_tests.cpp